src/modules/ekf2/ekf2_params.c

```diff
@@ -1567,3 +1567,56 @@ PARAM_DEFINE_INT32(EKF2_SYNT_MAG_Z, 0);
  * @decimal 1
  */
 PARAM_DEFINE_FLOAT(EKF2_GSF_TAS, 15.0f);
+
+
+
+
+/**
+ * Aux global position (AGP) sensor aiding
+ *
+ * Set bits in the following positions to enable:
+ * 0 : Horizontal position fusion
+ * 1 : Vertical position fusion
+ * 2 : Yaw
+ *
+ * @group EKF2
+ * @min 0
+ * @max 15
+ * @bit 0 Horizontal position
+ * @bit 1 Vertical position
+ * @bit 2 Yaw
+ */
+PARAM_DEFINE_INT32(EKF2_AGP_CTRL, 1);
```

Comment:
Can we remove the info for bit 2 if we're now using the message that does not support yaw/heading?